### DISCUSSION TITLE
crops_coords_top_left in the train_controlnet_sdxl.py why not use the actual value but set the value?

### DISCUSSION THREAD

**Discussion by Tramac** [huggingface/diffusers#7687] [Q&A]
I'm curious why `crops_coords_top_left` in the [train_controlnet_sdxl.py](https://github.com/huggingface/diffusers/blob/main/examples/controlnet/train_controlnet_sdxl.py#L1024) is not used with actual values but set values.

same as `original_size`.

```
original_size = (args.resolution, args.resolution)
target_size = (args.resolution, args.resolution)
crops_coords_top_left = (args.crops_coords_top_left_h, args.crops_coords_top_left_w)
```

**Comment by tolgacangoz** [comment]
Isn't it to be able to adjust [crop conditioning](https://huggingface.co/docs/diffusers/main/en/using-diffusers/sdxl#crop-conditioning)?
